yapf/yapflib/pytree_unwrapper.py

```diff
@@ -333,7 +333,8 @@ def _DetermineMustSplitAnnotation(node):
   if not _ContainsComments(node):
     token = next(node.parent.leaves())
     if token.value == '(':
-      if sum(1 for ch in node.children
+      if sum(1
```

Comment:
This change has be a little concerned. I know it's a corner case, but I find this new version to be less readable. Perhaps you could not split when the variable is a single integer or name? What are your thoughts?

Reply:
Yeah I was thinking the same thing. I think the one-token expression is a good idea. Now if only Python allowed `len(generator)`...